app/src/main/res/values-de/strings-de.xml

```diff
@@ -480,13 +480,310 @@
 	<string name="predictive_model_inner_workings">Vorhersagemodell (Erweiterte Anzeige)</string>
 	<string name="noise_model_inner_workings">Rauschmodell (Erweiterte Anzeige)</string>
 	<string name="glucose_number_from_filtered">Glukose aus gefilterten Werten</string>
-	<string name="if_the_phone_has_text_to_speech">Sofern das Telefon über Text-in-Sprache verfügt, wird jeder neue Wert vorgelesen.</string>
+	<string name="if_the_phone_has_text_to_speech">Sofern das Telefon über Text-zu-Sprache verfügt, wird jeder neue Wert vorgelesen.</string>
```

Comment:
Google calls this "Text-in-Sprache". The other sounds more idiomatic to me as well, but I'd stick with the name the user can find in the phone's settings.